src/app/app-routing.module.ts

```diff
@@ -3,7 +3,6 @@ import { SignUpComponent } from './component/user/auth/sign-up/sign-up.component
 import { AuthComponent } from './component/user/auth/auth.component';
 import { SignInComponent } from './component/user/auth/sign-in/sign-in.component';
 import { SubmitEmailComponent } from './component/user/auth/submit-email/submit-email.component';
```

Comment:
general: please move all components which are used in the map module to the map directory

Reply:
fixed